airdrop/infrastructure/models.py

```diff
@@ -105,7 +108,6 @@ class AirdropWindowTimelines(Base, AuditClass):
     date = Column("date", TIMESTAMP(), nullable=False)
```

Comment:
This column name is very generic

Reply:
Already existing column, will let Rajeev look into this

Reply:
The date column is on the AidropWindowTimeline table, which represents the timeline date. Do we need to rename it?